projects/static/projects/js/members.js

```diff
@@ -0,0 +1,28 @@
+function resetRole(selector_id) {
```

Comment:
nit: in Javascript, snake_case is almost never used, rather, it would be camelCase.